go/api/commands.go

```diff
@@ -647,3 +647,281 @@ type HashCommands interface {
 	// [valkey.io]: https://valkey.io/commands/hstrlen/
 	HStrLen(key string, field string) (Result[int64], error)
 }
+
+// SetCommands supports commands and transactions for the "Set Commands" group for standalone and cluster clients.
+//
+// See [valkey.io] for details.
+//
+// [valkey.io]: https://valkey.io/commands/?group=set
+type SetCommands interface {
```

Comment:
Let's move this interface in a separate file to keep the command definitions in dedicated files.

Reply:
@janhavigupta007 now, It is separated.